lib/buildingsync/makers/model_maker_level_zero.rb

```diff
@@ -58,7 +58,7 @@ def generate_baseline(dir, epw_file_path, standard_to_be_used)
            end
       end
 
-      @facilities[0].generate_baseline_osm(epw_file_path, standard_to_be_used)
+      @facilities[0].generate_baseline_osm(epw_file_path, dir, standard_to_be_used)
```

Comment:
is there a guard to ensure that there will only be 1 facility?

Reply:
yes, just above we check the facility count